.github/workflows/issues.yml

```diff
@@ -39,17 +37,14 @@
                       "id": "12"
                   },
                   "customfield_12751": [{
-                          "id": "22222"
-                  }],
+                          "id": "25550"
+                  }],                  
                   "description": "This ticket tracks the following GitHub issue: '"${ISSUE_URL}"'.",
                   "components": [
                       {
-                          "id": "32194"
+                          "id": "34035"
```

Comment:
Do we want to change component?

Reply:
@lantoli not sure about this one - looking for answer before approval

Reply:
you're totally right, let's keep it, change here https://github.com/mongodb/atlas-sdk-go/pull/437/commits/3ae779f1c6fce17d41aaed55047fa9968d2d0e23